client/src/modules/shopping-list/model/selectors.js

```diff
@@ -4,15 +4,18 @@ import _keyBy from 'lodash/keyBy';
 
 export const getList = (state, listId) =>
   _pick(state.lists.data, listId)[listId];
-export const getItemsForCurrentList = (state, listId) => {
-  const currentList = _pick(state.lists.data, listId)[listId];
-  if (
-    currentList &&
-    Object.prototype.hasOwnProperty.call(currentList, 'products')
-  ) {
+export const getItems = (state, listId) => {
+  const currentList = getList(state, listId);
```

Comment:
wonder if we need to use currentList? maybe 'list' is enough?

Reply:
in my opinion currentList is a good variable in this circumstance